utils/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 
 [dependencies]
 reqwest = { version = "0.11", features = ["json"] } 
-libnixstore = { git = "https://github.com/nix-community/harmonia.git" }
+libnixstore = { git = "https://github.com/JulienMalka/harmonia.git" }
```

Comment:
I think the fix has been merged upstream now!

Reply:
hmm, current upstream doesn't seem to compile :/